week-3/implement/get-card-value.js

```diff
@@ -29,3 +29,24 @@
 // Given a card with an invalid rank (neither a number nor a recognized face card),
 // When the function is called with such a card,
 // Then it should throw an error indicating "Invalid card rank."
+
+function getCardValue(card) {
+  const cardRank = card.slice(0, -1);
+
+  if (cardRank === "A") {
+    return 11;
+  } else if (cardRank === "K" || cardRank === "Q" || cardRank === "J") {
+    return 10;
+  } else {
+    if (!isNaN(cardRank)) {
+      return parseInt(cardRank);
+    } else {
+      throw new Error("Invalid card rank");
+    }
+  }
```

Comment:
I learnt something new here. I did not know what !isNan and parseInt and throw new Error mean. But I looked up on MDN website and found the answer. 
That is a pretty good code. Well done!